src/monovec.jl

```diff
@@ -20,7 +20,7 @@ Calling `monovec` on ``[xy, x, xy, x^2y, x]`` should return ``[x^2y, xy, x]``.
 """
 function monovec(X::AbstractVector{MT}) where {MT<:AbstractMonomial}
     Y = sort(X, rev=true)
-    dups = find(i -> Y[i] == Y[i-1], 2:length(Y))
+    dups = findall(i -> Y[i] == Y[i-1], 2:length(Y))
```

Comment:
How does it work in Julia v0.6.2, when I write `findall` in Julia v0.6.2, it says that it is not defined but this passes on travis. This line is probably not executed on Travis as it is only used with TypedPolynomials